primefaces/src/main/java/org/primefaces/component/datatable/export/DataTablePDFExporter.java

```diff
@@ -161,6 +161,12 @@ protected void addEmptyLine(Paragraph paragraph, int number) {
         }
     }
 
+    protected void addCell(PdfPTable table, PdfPCell cell) {
+        if (table != null && cell != null) {
+            table.addCell(cell);
+        }
```

Comment:
Those null check are really necessary? if they're null there is a truly a problem

Reply:
Removed.  I was just being defensive they will never be null.